### ISSUE TITLE
counterbalancing choice-option allocation across trials

### ISSUE THREAD

**Issue by SantiGarciaG** [SantiGarciaG/iswsIII#2]
If I want to randomise the choice option position, not from block to block, but from trial to trial, if it just a matter of moving codeline 25, to somewhere in before codeline 116?

**Comment by cherepaha** [comment]
Yep, but don't forget that it is passed as an argument through run_block() to run_trial(). So after moving the randomisation to within run_trial, you'd need to clear it up properly at the higher levels

**Comment by SantiGarciaG** [comment]
I keep missing something here, but in any case, after going over things with Denis, he is strongly suggest to put a constrain in this randomization so the position of a specific selection option (left or right) is not repeated more than three consecutive times. I have no idea how to do this, so as always, you'll have to be my saviour.
